tests/foreman/cli/test_user.py

```diff
@@ -429,10 +445,26 @@ def test_positive_personal_access_token_user_with_role(self):
 
         :CaseImportance: High
         """
+        user = make_user()
```

Comment:
(Optional): Can we create a common fixture for user, role, and token?

Reply:
Keeping it as an improvement. Will be adding new scenarios as well